data/events.json

```diff
@@ -23,7 +23,7 @@
     },
     "book_club": {
       "meetup_url": "https://www.meetup.com/PhillyCocoaHeads/events/244904265/",
-      "date": "2017-11-21",
+      "date": "2017-12-05",
```

Comment:
Since this PR is getting reviewed after the 5th, this is already out of date so let's try to update it before the merge.